products/compute/api.yaml

```diff
@@ -935,6 +935,154 @@ objects:
         description: |
           How many seconds to wait for the backend before considering it a
           failed request. Default is 30 seconds. Valid range is [1, 86400].
+  - !ruby/object:Api::Resource
+    name: 'RegionBackendService'
+    kind: 'compute#backendService'
+    base_url: projects/{{project}}/regions/{{region}}/backendServices
+    collection_url_response: !ruby/object:Api::Resource::ResponseList
+      kind: 'compute#backendServicesList'
+      items: 'items'
+    has_self_link: true
+    description: |
+      A Region Backend Service defines a regionally-scoped group of virtual
```

Comment:
It's probably worth noting that RBS is only for ILB, like we did before: https://github.com/terraform-providers/terraform-provider-google-beta/pull/645/files#diff-83aa3ab4e2ca346150e6962897976dbaL15